src/core/qgsproject.cpp

```diff
@@ -66,6 +66,12 @@
 #include <utime.h>
 #endif
 
+static QString generateUuid()
+{
+  QString uuid = QUuid::createUuid().toString();
+  return uuid.mid( 1, uuid.length() - 3 );
```

Comment:
off topic - but wow it annoys me the format QUuid uses.

Reply:
You and be both. Never once have I thought "man I wish this guid had { } around it". Silly.